bin/utils.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# -------------------------------------------------------------
+
+import sys
+import os
+from os.path import join, exists
+from os import environ
+import shutil
+
+
+def get_env():
+    """
```

Comment:
Could you please give this function a more descriptive name. `get_env()` is too generic and doesn't convey that only `SPARK_HOME` and `SYSTEMML_HOME` variables are being returned. 
I would suggest you break this function up into two functions - one to get the environment variable for `SYSTEMML_HOME` and one for `SPARK_HOME`. You can then name them `get_env_systemml_home` and `get_env_spark_home` or something better.